src/str.js

```diff
@@ -716,6 +755,162 @@ Sk.builtin.str.prototype["isdigit"] = new Sk.builtin.func(function (self) {
     return Sk.builtin.bool(true);
 });
 
+Sk.builtin.str.prototype["isspace"] = new Sk.builtin.func(function (self) {
+    var ch;
+    var i;
+    var newstr;
+    Sk.builtin.pyCheckArgs("isspace", arguments, 1, 1);
+    if (self.v.length === 0) {
+        return Sk.builtin.bool(false);
+    }
+    newstr = goog.string.normalizeWhitespace(self.v);
```

Comment:
why not just use: `self.v.replace(/\xa0|\s/g, ' ');` I don't think we use the closure lib anywhere else.